### PR TITLE
Fix abs rubocop metric of create_activity

### PR DESCRIPTION
**Pull request by AllanKlaus**
Fixed Assignment Branch Condition size for create_activity is too high. [23.75/15]

https://codeclimate.com/github/chatwoot/chatwoot/app/models/conversation.rb#issue_5d87480c87cf1900010003c1

Issue: #54

### CODE REVIEW

File: app/models/conversation.rb [hunk 1]
```diff
@@ -4,11 +4,11 @@ class Conversation < ApplicationRecord
   validates :account_id, presence: true
   validates :inbox_id, presence: true
 
-  enum status: [ :open, :resolved ]
+  enum status: [:open, :resolved]
 
   scope :latest, -> { order(created_at: :desc) }
   scope :unassigned, -> { where(assignee_id: nil) }
-  scope :assigned_to, -> (agent) { where(assignee_id: agent.id) }
+  scope :assigned_to, ->(agent) { where(assignee_id: agent.id) }
```
Comment (pranavrajs) [inline app/models/conversation.rb]: Does this comply with rubocop?

**Comment by pranavrajs** [comment]
@AllanKlaus Thanks for the PR. It looks like there is a conflict with the base branch, could you please rebase it with master?

**Comment by AllanKlaus** [comment]
> @AllanKlaus Thanks for the PR. It looks like there is a conflict with the base branch, could you please rebase it with master?

already resolved

**Comment by AllanKlaus** [comment]
@pranavrajs 

the method on the master was this way

```ruby
  def create_activity
    return unless Current.user

    self.messages.create(activity_message_params(status_changed_message)) if status_changed?
    self.messages.create(activity_message_params(assignee_changed_message)) if assignee_id_changed?
  end
```

in my branch was this

```ruby
  def create_activity
    user_name = Current.user&.name
    # to prevent error when conversation is reopened by customer itself by sending a new message
    create_status_change_message(user_name) if status_changed? && Current.user
    create_assignee_change(username) if assignee_id_changed? && Current.user
  end
```

when I resolved the conflict become

```ruby
  def create_activity
    return unless Current.user
    user_name = Current.user&.name
    # to prevent error when conversation is reopened by customer itself by sending a new message
    create_status_change_message(user_name) if status_changed?
    create_assignee_change(username) if assignee_id_changed?
  end
```

**Pull request merged**